site/en/r2/tutorials/distribute/training_loops.ipynb

```diff
@@ -702,7 +702,22 @@
         "id": "MuZGXiyC7ABR"
       },
       "source": [
-        "Note: We do not use `tf.metrics.Mean` to track the training loss across different replicas because of how we scale the training loss. For example, let us assume that we have a global batch size of 4 and two devices. Two samples are processed on each of the devices. Let the loss values be [2,  3] and [4,  5] on each of the two devices. With loss scaling, the value of loss on each replica is 5/4=1.25 and 9/4=2.25. Note that we divide by 4 which is the global batch size. If we use `tf.metrics.Mean` to track loss we end up with a `total` of 3.50 and `count` of 2 which results in  `total`/`count`=1.75  when `result()` is called on the metric. The `tf.keras.Metrics` loss is scaled by an additional factor which is equal to the number of replicas in sync. As a general rule you should use `tf.keras.Metrics` to track per sample values and avoid values that have been aggregated within a replica."
+        "### Tracking training loss across replicas\n",
+        "\n",
+        "As a general rule, you should use `tf.keras.Metrics` to track per-sample values and avoid values that have been aggregated within a replica. Loss scaling with `tf.keras.Metrics` accounts for the number of replicas in sync.\n",
+        "\n",
+        "We do *not* recommend using `tf.metrics.Mean` to track the training loss across different replicas, because it does not account for the number of replicas in\n",
+        "sync.\n",
+        "\n",
+        "For example, if you run a training job with the following characteristics:\n",
+        "* Two replicas\n",
+        "* Two samples are processed on each replica\n",
+        "* Resulting loss values: [2,  3] and [4,  5] on each replica\n",
+        "* Global batch size = 4\n",
+        "\n",
+        "Loss calculated with `tf.keras.Metrics` is scaled by an additional factor that is equal to the number of replicas in sync. With loss scaling, you calculate the per-sample value of loss on each replica by adding the loss values, and then dividing by the global batch size. In this case: `(2 + 3) / 4 = 1.25` and `(4 + 5) / 4 = 2.25`. \n",
```

Comment:
It makes more sense for "Loss calculated with `tf.keras.Metrics` is scaled by an additional factor that is equal to the number of replicas in sync." to be the last line in this section.